### ISSUE TITLE
Installation instructions in README update

### ISSUE THREAD

**Issue by davidgil** [amitaibu/og#312]
Hi,

i am trying to install og but seems that installation instructions in README are obsolete. I cannot find in content type administration anything related to "Group type field". 

Best
David

**Comment by bunnyroo** [comment]
I agree. Some of the steps are no longer needed, others are missing like those in /config/group/. Any chance for updated instructions or is there a lot of change going on in the code we should wait for?
